<launch>

	<arg name="tracker_name" default="tracker" />
	<arg name="camera_frame" default="head_xtion_rgb_optical_frame"/>
	<arg name="pub_skeleton" default="True"/>
	<arg name="stand_alone" default="False"/>
    <arg name="log_skeleton" default="False"/>
    <arg name="message_store" default="people_skeleton"/>


	<node pkg="skeleton_tracker" type="xtion_tracker" name="skeleton_tracker" output="screen">
		<param name="tf_prefix" value="$(arg tracker_name)" />
    	<param name="relative_frame" value="head_xtion_depth_optical_frame" />
    	<param name="camera_frame" type="string" value="/$(arg camera_frame)"/>
	</node>




    <group if="$(arg pub_skeleton)">
	    <!-- Transforms /tf to /skeleton_data -->
	    <node pkg="skeleton_tracker" type="skeleton_publisher.py" name="skeleton_publisher" output="screen">
            <param name="log_skeleton" value="$(arg log_skeleton)" type="bool"/>
            <param name="message_store" value="$(arg message_store)" type="string"/>
	    </node>
    </group>


</launch>
